### PR TITLE
fix: [preview] preview picture error in 4K screen

### PR DESCRIPTION
**Pull request by GongHeng2017**
not get the right srceen.

Log: solved problem of preview plugin.
Bug: https://pms.uniontech.com/bug-view-192969.html https://pms.uniontech.com/bug-view-192975.html

### CODE REVIEW

**Review by max-lvs — DISMISSED** [review]

File: src/dfm-base/utils/universalutils.cpp [hunk 1]
```diff
@@ -449,4 +450,24 @@ void UniversalUtils::prepareForSleep(QObject *obj, const char *cslot)
             cslot);
 }
 
+QScreen *UniversalUtils::cursorScreen()
```
Comment (max-lvs) [inline src/dfm-base/utils/universalutils.cpp]: 在 UniversalUtils 还是 在WindowUtils 好呢？

Comment (GongHeng2017) [reply src/dfm-base/utils/universalutils.cpp]: 个人感觉UniversalUtils要好些，通用的接口。

**Review by max-lvs — APPROVED** [review]
+1

**Review by liyigang1 — APPROVED** [review]
[业务领域] 编码规范
[是否检查四性] Y 
[是否满足commit提交规范] Y 
[是否满足编码规范] Y 
[Review结论] Pass 
[Fail原因] N/A

**Pull request merged**